{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/utils/views.py

```diff
@@ -7,3 +18,36 @@ def get_template_names(self):
         template_name = self.kwargs.get('slug', '_base')
 
         return ['frontend/{}.html'.format(template_name)]
+
+
+class FrontendEditView(View):
+    def post(self, request):
+        if request.user.is_superuser:
+            data = json.loads(request.body.decode('utf-8'))
+            app_name = data.get('app', False)
+            model_name = data.get('model', False)
```

Comment:
Once again, if you are never expecting these to _not_ be present, this should fail with an exception (e.g. a KeyError when accessing `data['app']`). It also eliminates a branch below.